### DISCUSSION TITLE
Table sort

### DISCUSSION THREAD

**Discussion by m-y-n-o-n-a** [twbs/bootstrap#36175] [Ideas]
Columns with sort possibilities in tables would also be awesome

**Comment by coliff** [comment]
I think that might be out of the scope of Bootstrap. There are a number of feature-packed data table libraries - check out: https://datatables.net/examples/non_jquery/init.html for example. It can be used with Bootstrap.

Another good one is https://gridjs.io/.

**Reply by m-y-n-o-n-a** [reply]
Thank you for the recommendations but it would still be nice to have such basic behavior included instead of integrating another library
